can-value.md

```diff
@@ -4,6 +4,7 @@
 @collection can-core
 @description Get an observable that’s bound to a specific property on another object.
 @group can-value/methods methods
+@outline 2
```

Comment:
This makes the TOC look like:
<img width="334" alt="screen shot 2018-07-02 at 11 01 24 pm" src="https://user-images.githubusercontent.com/10070176/42201262-e2942e6a-7e4b-11e8-8d43-f77401824994.png">